ansible/roles/join-ad-linux/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+service_account_ssm_passwords:
+  service_account:
+    parameter: "/join_domain_linux_service_account/passwords"
```

Comment:
Longer term, we'll probably want to store other AD passwords in SSM or Secrets Manager.  Might be better to have a path like this
/ad_domain/azure.noms.root/join_domain_creds